epochStart/metachain/economics.go

```diff
@@ -455,14 +506,14 @@ func (e *economics) computeInflationRate(
 	supernovaInEpochActivated := e.enableEpochsHandler.IsFlagEnabledInEpoch(common.SupernovaFlag, prevEpoch)
 
 	if !supernovaInEpochActivated {
-		return e.computeInflationBeforeSupernova(computationData.GetRound(), prevEpoch)
+		return e.computeInflationBeforeSupernova(computationData.GetRound(), computationData.GetEpoch())
```

Comment:
the computationData epoch returns the new epoch, not the previous one.

I think here we need the previous epoch.